package.json

```diff
@@ -86,7 +85,7 @@
     "sass": "^1.46.0",
     "ts-jest": "^27.1.3",
     "tslib": "^1.10.0",
-    "typescript": "4.6.2"
+    "typescript": "^4.8.4"
```

Comment:
This is a large upgrade to jump 2 minor patches on typescript. Everything compiles correctly?

Reply:
Good call, I downgraded to 4.7.4 and that helped. Still a work in progress.